src/containers/TriggerBinding/TriggerBinding.js

```diff
@@ -0,0 +1,308 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import jsYaml from 'js-yaml';
+import './TriggerBinding.scss';
+import { injectIntl } from 'react-intl';
+import { InlineNotification, Tab, Tabs, Tag } from 'carbon-components-react';
+import { FormattedDate, Table } from '@tektoncd/dashboard-components';
+import {
+  getSelectedNamespace,
+  getTriggerBinding,
+  getTriggerBindingsErrorMessage,
+  isWebSocketConnected
+} from '../../reducers';
+
+import { fetchTriggerBinding } from '../../actions/triggerBindings';
+
+const initialState = {
+  showYAML: false,
+  showDetails: true
+};
+
+export /* istanbul ignore next */ class TriggerBindingContainer extends Component {
+  /* eslint-disable react/sort-comp */
+  state = initialState;
+
+  static notification({ kind, message, intl }) {
+    const titles = {
+      info: intl.formatMessage({
+        id: 'dashboard.triggerBinding.unavailable',
+        defaultMessage: 'TriggerBinding not available'
+      }),
+      error: intl.formatMessage({
+        id: 'dashboard.triggerBinding.errorloading',
+        defaultMessage: 'Error loading TriggerBinding'
+      })
+    };
+    return (
+      <InlineNotification
+        kind={kind}
+        hideCloseButton
+        lowContrast
+        title={titles[kind]}
+        subtitle={message}
+      />
+    );
+  }
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  componentDidUpdate(prevProps) {
+    const { match, webSocketConnected } = this.props;
+    const { namespace, triggerBindingName } = match.params;
+    const {
+      match: prevMatch,
+      webSocketConnected: prevWebSocketConnected
+    } = prevProps;
+    const {
+      namespace: prevNamespace,
+      triggerBindingName: prevTriggerBindingName
+    } = prevMatch.params;
+
+    if (
+      namespace !== prevNamespace ||
+      triggerBindingName !== prevTriggerBindingName ||
+      (webSocketConnected && prevWebSocketConnected === false)
+    ) {
+      this.fetchData();
+    }
+  }
+
+  fetchData() {
+    const { match } = this.props;
+    const { namespace, triggerBindingName } = match.params;
+    this.props.fetchTriggerBinding({ name: triggerBindingName, namespace });
+  }
+
+  handleTabSelect = index => {
```

Comment:
Carbon handles this for us. The content to be displayed should be a child of the corresponding Tab component. This is important for accessibility, ensuring the correct labels and navigation are used.

Reply:
ok so ill make the content look like Adam has in his Trigger details page PR